packages/mcp-provider-dx-core/test/e2e/deploy_metadata.test.ts

```diff
@@ -76,17 +71,20 @@ describe('deploy_metadata', () => {
 
     expect(result.isError).to.equal(false);
     expect(result.content.length).to.equal(1);
-    expect(result.content[0].type).to.equal('text');
+    if (result.content[0].type !== 'text') assert.fail();
```

Comment:
`expect(result.content[0].type).to.equal('text');` accomplishes the same but doesn't do type-checking so L76 would have `unknown` type.

The if conditional asserts that `result.content[0].text` is of type `string` (as defined in the mcp sdk)